core/src/com/unciv/logic/HexMath.kt

```diff
@@ -16,6 +14,18 @@ class HexMath {
         return getVectorForAngle((2 * Math.PI * (hour / 12f)).toFloat())
     }
 
+    fun getAdjacentVectors(origin: Vector2): ArrayList<Vector2> {
+        val vectors = ArrayList<Vector2>()
+        vectors += Vector2(1f, 0f)
+        vectors += Vector2(1f, 1f)
+        vectors += Vector2(0f, 1f)
+        vectors += Vector2(-1f, 0f)
+        vectors += Vector2(-1f, -1f)
+        vectors += Vector2(0f, -1f)
```

Comment:
this is already merged, but this is more kotlin way
`val vectors = listOf(
                Vector2(1f, 0f),
                Vector2(1f, 1f),
                Vector2(0f, 1f),
                Vector2(-1f, 0f),
                Vector2(-1f, -1f),
                Vector2(0f, -1f)
        )`

Reply:
I'm going to add this to the refactor of HexMath. Thanks